crates/framework/mos-framework/sources/account.move

```diff
@@ -1,8 +1,71 @@
 module mos_framework::account{
+   
+   use std::error;
+   use std::bcs;
+   use std::vector;
 
-   public fun create_account(_auth_key: vector<u8>): signer{
-      abort 0
+   /// Account already exists
+   const EACCOUNT_ALREADY_EXISTS: u64 = 1;
+   /// Account does not exist
+   const EACCOUNT_DOES_NOT_EXIST: u64 = 2;
+   /// Sequence number exceeds the maximum value for a u64
+   const ESEQUENCE_NUMBER_TOO_BIG: u64 = 3;
+   /// The provided authentication key has an invalid length
+   const EMALFORMED_AUTHENTICATION_KEY: u64 = 4;
+   /// Cannot create account because address is reserved
+   const ECANNOT_RESERVED_ADDRESS: u64 = 5;
```

Comment:
The error code styles should be:
The first letter is E, and the following words use camel case

Reply:
this is part of #26, please review it first, and I will make rebase.